### ISSUE TITLE
Quicktext version 3.4 doesn't work with Thunderbird version 78.6.0

### ISSUE THREAD

**Issue by jussiep** [thundernest/quicktext#221]
I just got back from holiday to find that Quicktext no longer works with the latest version of thunderbird. Can someone please help with this?  Many thanks.

**Comment by SamuelPlentz** [comment]
Could you provide additional information?
1. Is Quicktext still visible in the Addon dialog?
2. Could you reach Quicktext settings? (by pressing F10 to open Thunderbird menu bar > select Extras > select Quicktext)
3. If yes could you create a new template to insert "abc" into your E-Mail?

Does 1, 2, 3 work for you?

**Comment by SamuelPlentz** [comment]
Could you try if deactivating every other addon works.

**Comment by marcelosv2003** [comment]
Same issue here. Thunderbird 78.6.1 32-bit (Quicktext is incompatible with Thunderbird 78.6.1.)

**Comment by SamuelPlentz** [comment]
Strange it works fine here,

Could you answer the questions?

1. Is Quicktext still visible in the Addon dialog?
2. Could you reach Quicktext settings? (by pressing F10 to open Thunderbird menu bar > select Extras > select Quicktext)
3. If yes could you create a new template to insert "abc" into your E-Mail?
4. Could you try if deactivating every other addon works.
5. What OS do you use?

**Comment by marcelosv2003** [comment]
> 
> 
> Strange it works fine here,
> 
> Could you answer the questions?
> 
>     1. Is Quicktext still visible in the Addon dialog?
> 
>     2. Could you reach Quicktext settings? (by pressing F10 to open Thunderbird menu bar > select Extras > select Quicktext)
> 
>     3. If yes could you create a new template to insert "abc" into your E-Mail?
> 
>     4. Could you try if deactivating every other addon works.
> 
>     5. What OS do you use?

Hi Samuel.

1. Is Quicktext still visible in the Addon dialog?
Yes, but as Diasabled add-on (qt_disabled.jpg)
![qt_disabled](https://user-images.githubusercontent.com/77411979/104815624-73b7e380-57f4-11eb-8ec1-a97fd9a3333f.jpg)

2. Could you reach Quicktext settings? (by pressing F10 to open Thunderbird menu bar > select Extras > select Quicktext)
No

3. If yes could you create a new template to insert "abc" into your E-Mail?
Doesn't work (can't reach Quicktext)

4. Could you try if deactivating every other addon works.
Still not works (qt_all_add_on_disabled.jpg)
![qt_all_add_on_disabled](https://user-images.githubusercontent.com/77411979/104821618-f2744700-581b-11eb-87e9-c24587e68d69.jpg)

5. What OS do you use?
Windows 10 Pro
Version 1909
OS Build 18363.1256

Attached two more files:
- qt_err_install.jpg
- qt_incompatible.jpg

![qt_incompatible](https://user-images.githubusercontent.com/77411979/104821704-7c241480-581c-11eb-8993-87a0d288407b.jpg)
![qt_err_install](https://user-images.githubusercontent.com/77411979/104821706-7cbcab00-581c-11eb-8179-0b979cd31fc1.jpg)

And there are 5 entries related to Quicktext in the Invalidprefs.js file:
Search "quicktext" (5 hits in 1 file of 1 searched)
  C:\Users\NNNNNN\Downloads\Invalidprefs.js (5 hits)
	Line 179: user_pref("extensions.quicktext.collapseState", "1;1;1");
	Line 833: user_pref("quicktext.collapseState", "");
	Line 834: user_pref("quicktext.firstTime", false);
	Line 835: user_pref("quicktext.menuCollapse", false);
	Line 836: user_pref("quicktext.popup", true);

Regards,

**Comment by jobisoft** [comment]
You are you using version 1.1 (which is 2 years old). That is if course not compatible with TB78. Please remove that and than install v3.5.

**Comment by SamuelPlentz** [comment]
@jussiep Could you answer the questions?

**Comment by jussiep** [comment]
@SamuelPlentz so sorry for the delay in getting back to you... my partner suffered a serious back injury just after I logged this fault and I have not spent much time on the computer since. The issue occurred immediately after a Thunderbird upgrade and after I had closed Thunderbird and re-opened it a couple of times Quicktext started working again.  I had exactly the same issue on another computer as well.

Keep up the great work with this awesome add-on... it's a huge timesaver for me 🥇
